mlrun/model_monitoring/applications/base.py

```diff
@@ -902,13 +902,13 @@ def evaluate(
                                   if ``endpoints`` are passed.
                                   Note: the model monitoring infrastructure must be up for the writing to work.
         :param existing_data_handling:
-                                  How to handle the existing application data for the model endpoints when writing the
-                                  new data. Relevant only when ``write_output=True``. The default is
-                                  ``"fail_on_overlap"``. The options are:
+                                  How to handle the existing application data for the model endpoints when writing
+                                  new data whose requested `start` time precedes the `end `time of a previous run
```

Comment:
Double ` + fixed a space:

```suggestion
                                  new data whose requested ``start`` time precedes the ``end`` time of a previous run
```